P01 - Collision Detection/AppClassMain.cpp

```diff
@@ -496,4 +500,52 @@ void Simplex::Application::EndGame(void) {
 	else {
 		m_sEndGameMessage = "You Lost!";
 	}
+}
+
+void Simplex::Application::BulletShoot(void) {
```

Comment:
Good effort on refactoring the bullet shoot! However I think we should move this into the Bullet itself, so that we can have an array of bullet if we chose to and let them move themselves, instead of having the main program having to handle the moving logic.

Reply:
Another problem with this approach is that the fTimer is now a global variable, which we want to avoid. fTimer should be calculated once and then passed down to the bullet, instead of having the bullet recalculate the timer.